notebooks/samples/optimizer/ai_platform_optimizer_multi_objective.ipynb

```diff
@@ -0,0 +1,695 @@
+{
```

Comment:
<p>Add double line between imports, CONSTANTS and functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>